hs_core/hydroshare/utils.py

```diff
@@ -1075,6 +1075,16 @@ def check_aggregations(resource, folders, res_files):
                 # need relative folder path for creating aggregation from folder
                 fol = fol[len(resource.file_path) + 1:]
             agg_type = resource.get_folder_aggregation_type_to_set(folder)
+
+            if agg_type == 'TimeSeriesLogicalFile':
+                # check if the folder (fol) contains a csv file
+                res_files = ResourceFile.list_folder(resource=resource, folder=fol,
+                                                     sub_folders=False)
+                # there can be only one file in the folder
+                # if that file is a csv file - don't use the folder to create aggregation
+                if res_files[0].extension.lower() == '.csv':
+                    continue
+
```

Comment:
do we care about tsv or if it's a csv but improperly named?

Reply:
I think the requirement is that the file should have the extension of 'csv'. @horsburgh can comment further.

Reply:
Understood.